recipes/distopia/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "distopia" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/distopia-{{ version }}.tar.gz
+  sha256: 9118b6c6bf1589be3f7d8be0e295bdf737a9e2a13c54e6fac28f9aea8e50bf51
+
+build:
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - numpy
+    - scikit-build
+    - cython
+    - versioneer-518
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - distopia
+  requires:
+    - pip
+    - cmake
```

Comment:
@xhochy I'm not sure what I'm meant to be doing with the requirments in this section as its complaining unless I add some of these deps in. 

I also have cmake here and it looks like its not picking it up?

Thanks for all your help.

Reply:
They are listed as a requirement here: https://github.com/MDAnalysis/distopia/blob/63ebe4b73f9fcb152f7ee4dea116a7a0b990cc11/setup.py#L51

Either they should be in `run` or if not needed at runtime, this should be patched out.

Reply:
These have been patched out now, apologies for the confusion.

Reply:
This has been patched out now apologies for confusion.